syne_tune/backend/local_backend.py

```diff
@@ -229,7 +229,7 @@ def _all_trial_results(self, trial_ids: List[int]) -> List[TrialResult]:
             res.append(trial_results)
         return res
 
-    def _pause_trial(self, trial_id: int):
+    def _pause_trial(self, trial_id: int, result: Optional[dict]):
```

Comment:
there are multiple cases where the typing suggests optional but `None` was not provided as default value.

Reply:
These are internal methods, you'd not call them directly.